src/lib/structures/Inhibitor.js

```diff
@@ -32,6 +32,22 @@ class Inhibitor extends Piece {
 		this.spamProtection = options.spamProtection;
 	}
 
+	/**
+	 * The async wrapper for running inhibitors
+	 * @since 0.5.0
+	 * @param {KlasaMessage} message The message that triggered this inhibitor
+	 * @param {Command} command The command to run
+	 * @returns {(void|string)}
+	 * @private
+	 */
+	async _run(message, command) {
```

Comment:
Oh yeah, this is way better as it allows better customization (like making our own Inhibitor class extending this one to always log `Error` instances in an `inhibitorError` event), just typings and this should be fine to merge 👍

Reply:
Also, should we make this `protected` in typings? So we can override its behaviour. Not sure if you want to change `@private` to `@protected`.

Reply:
not sure if protected is handled by our jsdocs/website (atleast properly). It can be protected in typings.

Reply:
No, they aren't handled at the moment. `Store#registerCoreDirectory` is marked with `@protected`. https://klasa.js.org/#/docs/klasa/master/class/Store?scrollTo=registerCoreDirectory but the documentation does not show any label for it.